packages/generator/templates/app/test/index.test.tsx

```diff
@@ -28,10 +28,9 @@
       })
     }
   ))
- 
+
    const { getByText } = render(<Home />)
-   const linkElement = getByText(/Documentation/i)
+   const linkElement = getByText(/Blitz Docs/i)
```

Comment:
Is this the right place to change this? Using my local dev version of blitz, I couldn't manage to create a new app with the updated template.

Also, should this file be updated in `apps/toolkit-app` too?

Reply:
yes @jhonnymichel, it's the correct location for the change.

Yeah you could update the files in `apps/toolkit` with the new design and test